custom/icds/location_reassignment/parser.py

```diff
@@ -79,6 +83,11 @@ def _parse_row(self, row, location_type_code):
         if self._invalid_row(operation, old_site_code, new_site_code):
             return
         self._note_transition(operation, location_type_code, new_site_code, old_site_code)
+        self.new_location_details[location_type_code].append({
+            'name': row.get(NEW_NAME), 'site_code': new_site_code,
```

Comment:
whitespace: one dict property per line

Reply:
something we follow as a standard?

Reply:
I think so. We're inconsistent about parameter lists, but I think dicts are either a property per line or the whole dict on one line (rarely, for small dicts).